src/rtags.el

```diff
@@ -1234,8 +1232,12 @@ absolute-location to remote. absolute-location can of course be a path"
           (tramp-tramp-file-p absolute-location))
       absolute-location
     (let ((location-vec (tramp-dissect-file-name default-directory)))
-      (aset location-vec 3 absolute-location)
-      (apply 'tramp-make-tramp-file-name (append location-vec nil)))))
+      (if (arrayp location-vec)
```

Comment:
Old Tramp compatibility

Reply:
> Those this work with older Emacs versions as well e.g, 24.3.

Yes I added a compatibility conditional for older Tramp releases (still using array representation)